packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -18,7 +18,6 @@ export const disableInputAttributeSyncing = __VARIANT__;
 export const enableFilterEmptyStringAttributesDOM = __VARIANT__;
 export const enableLegacyFBSupport = __VARIANT__;
 export const skipUnmountedBoundaries = __VARIANT__;
-export const enableSuspenseLayoutEffectSemantics = __VARIANT__;
```

Comment:
just curious, what's the purpose of the `__VARIANT__` value?

Reply:
For dynamic flags, we use the `__VARIANT__` setting to ensure that automated tests run with both values (enabled and disabled) so we don't accidentally break one.